test/views/RefundToTaxPayerViewSpec.scala

```diff
@@ -0,0 +1,272 @@
+    val tableHeadRequestedAmount: String = messages("refund-to-taxpayer.tableHead.requested-amount")
+    val tableHeadRefundAmount: String = messages("refund-to-taxpayer.tableHead.refund-amount")
+    val tableHeadInterest: String = messages("refund-to-taxpayer.tableHead.interest")
+    val tableHeadTotalInterest: String = messages("refund-to-taxpayer.tableHead.total-interest")
+    val variedInterest: String = s"${messages("refund-to-taxpayer.tableHead.total-interest")} ${messages("refund-to-taxpayer.tableHead.varied-interest-rates-value", "1.76", "2.01", "31 July 2021", "30 November 2021")}"
+
+    // todo check if it's needed
+    //    val partialH2Heading = "payments"
+    //    val saLink: String = s"${messages("whatYouOwe.sa-link")}${messages("pagehelp.opensInNewTabText")}"
+  }
+
+  val testRepaymentHistoryModel: RepaymentHistoryModel = RepaymentHistoryModel(
+    List(RepaymentHistory(
+      Some(705.2),
+      705.2,
+      "BACS",
+      12345,
+      Vector(
+        RepaymentItem(
+          Vector(
+            RepaymentSupplementItem(
+              Some("002420002231"),
+              Some(3.78),
+              Some(LocalDate.of(2021, 7, 31)),
+              Some(LocalDate.of(2021, 9, 15)),
+              Some(2.01)
+            ),
+            RepaymentSupplementItem(
+              Some("002420002231"),
+              Some(2.63),
+              Some(LocalDate.of(2021, 9, 15)),
+              Some(LocalDate.of(2021, 10, 24)),
+              Some(1.76)
+            ),
+            RepaymentSupplementItem(
+              Some("002420002231"),
+              Some(3.26),
+              Some(LocalDate.of(2021, 10, 24)),
+              Some(LocalDate.of(2021, 11, 30)),
+              Some(2.01))
+          )
+        )
+      ), LocalDate.of(2021, 7, 23), LocalDate.of(2021, 7, 21), "000000003135")
+    )
+  )
+
+  val testRepaymentHistoryModelRequestedAmountDiffersToRefundAmount: RepaymentHistoryModel =
+    testRepaymentHistoryModel.copy(
+      List(
+        testRepaymentHistoryModel.repaymentsViewerDetails.head.copy(amountApprovedforRepayment = Some(800.12), amountRequested = 345.5))
+    )
+
+  val testRepaymentHistoryModelRequestedMissingRefundAmount: RepaymentHistoryModel =
+    testRepaymentHistoryModel.copy(
+      List(
+        testRepaymentHistoryModel.repaymentsViewerDetails.head.copy(amountApprovedforRepayment = None, amountRequested = 345.5))
+    )
+
+  class RefundToTaxPayerViewSetup(testRepaymentHistoryModel: RepaymentHistoryModel, saUtr: Option[String] = Some("AY888881A"), isAgent: Boolean = false) extends Setup(
+    refundToTaxPayerView(testRepaymentHistoryModel, paymentHistoryRefundsEnabled = false, "testBackURL", saUtr, isAgent = isAgent)(FakeRequest(), implicitly)
+  )
+
+  //  val paymentHistoryMessageInfo = s"${messages("paymentHistory.info")} ${messages("taxYears.oldSa.agent.content.2")}${messages("pagehelp.opensInNewTabText")}. ${messages("paymentHistory.info.2")}"
```

Comment:
Please update to remove the commented line

Reply:
Thanks, Done